packages/wallets/src/features/cfd/flows/Verification/Verification.tsx

```diff
@@ -357,6 +357,7 @@ const Verification: FC<TVerificationProps> = ({ selectedJurisdiction }) => {
 
                 return (
                     <ModalStepWrapper
+                        disableScroll={context.currentScreenId === 'personalDetailsScreen'}
```

Comment:
To prevent scrolling in that specific modal, ensure `overflow: auto` is used only when content overflows. In this modal, the dynamic dropdown content triggers scrolling due to its collapsible nature. This is currently the only modal experiencing this issue.

Reply:
https://github.com/binary-com/deriv-app/assets/103104395/cecf8fbd-9789-4d1c-ad31-39b11e848728

Reply:
Also FYI I have reduce the list height as well in this PR so it won't touch go all the way down the screen.